src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -164,11 +166,14 @@ public static ReadOnlyPropertyBook getSamplePropertyBook() {
     public static Meeting[] getSampleMeetings() {
         return new Meeting[] {
             new Paperwork(new BidderId("B1"), new PropertyId("P2"),
-                        new Time("15 OCT 2020"), new Venue("Marina Bay")),
+                        new Date("15 OCT 2020"), new Venue("Marina Bay"),
```

Comment:
The format of the date object should be dd-MM-YYYY instead. pls note that this might affect future commands that the user inputs such as the sort meeting command. Else, it lgtg.